TypeError in _get_potcar_symbols()

`class Lobsterin()._get_potcar_symbols()` causes [CI errors in `atomate`](https://github.com/hackingmaterials/atomate/runs/4871532128?check_suite_focus=true#step:8:115)

https://github.com/materialsproject/pymatgen/blob/fd553a8e80935d77df285fed1a14d7bf63669c14/pymatgen/io/lobster/inputs.py#L612-L633

```sh
        with zopen(POTCAR_input, "r") as f:
            data = f.read()
>       if "SHA256" in data or "COPYR" in data:
E       TypeError: a bytes-like object is required, not 'str'
```

Triggered by PR: https://github.com/hackingmaterials/atomate/pull/738

--- a/pymatgen/io/lobster/inputs.py
+++ b/pymatgen/io/lobster/inputs.py
@@ -617,6 +617,8 @@ class Lobsterin(dict, MSONable):
         # Warning about a bug in lobster-4.1.0
         with zopen(POTCAR_input, "r") as f:
             data = f.read()
+        if isinstance(data, bytes):
+            data = data.decode("utf-8")
         if "SHA256" in data or "COPYR" in data:
             warnings.warn(
                 "These POTCARs are not compatible with "

